verify/boilerplate/boilerplate.py

```diff
@@ -86,7 +86,7 @@ def file_passes(filename, refs, regexs):
         (data, found) = p.subn("", data, 1)
 
     # remove shebang from the top of shell files
-    if extension == "sh":
+    if extension == "sh" or extension == "py":
```

Comment:
The enforcement of `/usr/bin/env` needs to be done. Maybe it doesn't belong in the license check, but it should be done...